packages/schema-blocks/src/functions/gutenberg/sidebar.ts

```diff
@@ -0,0 +1,15 @@
+import { select, dispatch } from "@wordpress/data";
+
+/**
+ * Opens the given sidebar unless the given sidebar or another sidebar is already open.
```

Comment:
Not catched by the linter, but I would love to see a newline after this.

```suggestion
 * Opens the given sidebar unless the given sidebar or another sidebar is already open.
 *
```